deployments/container/reconfigure-mig.sh

```diff
@@ -52,6 +54,8 @@ function usage() {
   echo "    -p <default-gpu-clients-namespace>            Default name of the Kubernetes Namespace in which the GPU client Pods are installed in"
   echo "    -t <driver-root>                              Root path to the NVIDIA driver installation"
   echo "    -a <driver-root-ctr-path>                     Root path to the NVIDIA driver installation mounted in the container"
+  echo "    -b <dev-root>                                 Root path to the NVIDIA device nodes"
```

Comment:
Corresponding options would also need to be added to the mig-manager go binary which invokes this shell script.